flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackend.java

```diff
@@ -281,6 +281,12 @@ public StateBackend getCheckpointBackend() {
         return checkpointStreamBackend;
     }
 
+    @Override
+    public boolean supportsForcedFullSnapshot() {
+        // we respect the flag and potentially reupload shared files
```

Comment:
I don't think this method would accept any flag, we should describe it as whether be able to do so.